client/dashboard/sites/overview/storage-warning-banner.tsx

```diff
@@ -0,0 +1,66 @@
+import { useSuspenseQuery, useMutation } from '@tanstack/react-query';
+import { sprintf, __ } from '@wordpress/i18n';
+import filesize from 'filesize';
+import { userPreferenceQuery, userPreferenceMutation } from '../../app/queries/me-preferences';
+import { siteMediaStorageQuery } from '../../app/queries/site-media-storage';
+import Notice from '../../components/notice';
+import UpsellCTAButton from '../../components/upsell-cta-button';
+import { getStorageAlertLevel } from '../../utils/site-storage';
+import type { Site } from '../../data/types';
+
+export function StorageWarningBanner( { site }: { site: Site } ) {
+	const { data: mediaStorage } = useSuspenseQuery( siteMediaStorageQuery( site.ID ) );
+	const { data: isDismissedPersisted } = useSuspenseQuery(
```

Comment:
Should we ensure the data in the routes, given that we're using `useSuspenseQuery()` here?

Reply:
Whoops I mis-read the router code. The preferences are already ensure'd in the *sites* route, not the *site* route 🤦 
Done in f335f330dc0

When I test by deleting the `REACT_QUERY_OFFLINE_CACHE` object I really am getting a clean loading experience now.